client/src/pages/learn/coding-interview-prep/take-home-projects/index.md

```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-This introduction is a stub
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete in their own time. These type of interviews usually require a lot of work but it is the most preferred interviewing process.
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
+
+In this section you get the chance to work on many Take Home Projects like:
+*  Show Local Weather
+*  Build a Wikipedia Viewer
+*  Build a Recipe Box and many more...
+
+By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and make you prepared for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

Comment:
```suggestion
By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```